drivers/bluetooth/hci/ipc.c

```diff
@@ -22,6 +22,14 @@ LOG_MODULE_REGISTER(bt_hci_driver);
 
 #define IPC_BOUND_TIMEOUT_IN_MS K_MSEC(1000)
 
+/* The retry of ipc_service_send function requires a small (tens of us) delay.
```

Comment:
```suggestion
/* The retry of ipc_service_send function may require a delay.
```
There's no maximum of `BT_HCI_IPC_SEND_RETRY_DELAY_US` so it could be configured to be e.g.  10 seconds. Suggest not to use terms like `small` or `tens of us` unless we want to add limits to the Kconfig option

Reply:
There isn't a limit but testing shows that the controller processing times are in 30-90 us range. So using tens of us is a good starting point.

Reply:
> ipc_service_send function requires a small (tens of us) delay

It's fairly obvious to notice that function requires small delay so user should put small number. If user puts large number this is not what the function requires.